custom/icds_reports/tests/outputs/agg_ccs_record_2017-05-01_3_sorted.csv

```diff
@@ -1,15 +1,15 @@
 state_id,district_id,block_id,supervisor_id,awc_id,month,ccs_status,trimester,caste,disabled,minority,resident,valid_in_month,lactating,pregnant,thr_eligible,rations_21_plus_distributed,tetanus_complete,delivered_in_month,anc1_received_at_delivery,anc2_received_at_delivery,anc3_received_at_delivery,anc4_received_at_delivery,registration_trimester_at_delivery,using_ifa,ifa_consumed_last_seven_days,anemic_normal,anemic_moderate,anemic_severe,anemic_unknown,extra_meal,resting_during_pregnancy,bp1_complete,bp2_complete,bp3_complete,pnc_complete,trimester_2,trimester_3,postnatal,counsel_bp_vid,counsel_preparation,counsel_immediate_bf,counsel_fp_vid,counsel_immediate_conception,counsel_accessible_postpartum_fp,has_aadhar_id,aggregation_level,valid_all_registered_in_month,institutional_delivery_in_month,lactating_all,pregnant_all,valid_visits,expected_visits,state_is_test,district_is_test,block_is_test,supervisor_is_test,awc_is_test
 st1,d1,b1,All,All,2017-05-01,lactating,All,All,All,All,All,43,43,0,43,5,0,2,0,0,0,0,3.0000000000000000,0,0,0,0,1,42,0,0,2,2,3,1,0,0,6,0,0,0,0,0,0,18,3,43,1,43,0,0,53,0,0,0,0,0
-st1,d1,b1,All,All,2017-05-01,other,All,All,All,All,All,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,195,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,55,3,0,0,0,0,0,80.93,0,0,0,0,0
+st1,d1,b1,All,All,2017-05-01,other,All,All,All,All,All,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,8,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,3,0,0,0,0,0,8,0,0,0,0,0

```

Comment:
how come some of these appear to change from a decimal to int

Reply:
Because these were evaluated to be a whole number. This has happened because of some rows are deleted which are ccs_status='Others' which were contributing with 0.39 in this column(expected_visits)